TraderJoe-Bots/Contract-Ownership/src/findings.ts

```diff
@@ -0,0 +1,17 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+
+export const createFinding = (name: string, args: any, logAddress: string) => {
+  return Finding.fromObject({
+    name: "Ownership of a monitored contract has changed",
+    description: `${name} event was emitted`,
```

Comment:
Maybe this is better for a finding description? 

```suggestion
    name: "Ownership of a TraderJoe contract has changed",
    description: `${name} event was emitted`,
```